lib/src/file_picker_macos.dart

```diff
@@ -90,31 +79,27 @@ class FilePickerMacOS extends FilePicker {
     Uint8List? bytes,
     bool lockParentWindow = false,
   }) async {
-    final String executable = await isExecutableOnPath('osascript');
-    final String fileFilter = fileTypeToFileFilter(
+    assert(bytes == null, "Passing bytes is not supported on macOS.");
```

Comment:
Can we use an `UnsupportedError` instead? I wonder how we do this for other platforms (i.e. web)

Reply:
done